lib/ddtrace/contrib/extensions.rb

```diff
@@ -35,14 +35,14 @@ def registry
 
       # Configuration methods for Datadog module.
       module Configuration
-        def configure(target = configuration, opts = {})
+        def configure(configuration = self.configuration, opts = {})
           # Reconfigure core settings
           super
```

Comment:
Aside: I vaguely remember discussing this before but, is it time to move this to the core (as a separate step, not in this PR)? The registry is something that I think makes sense in the core.

Reply:
Makes sense, I added a `TODO` comment inline there so we don't forget this.